modules/geospatial/src/type-utils.js

```diff
@@ -56,3 +57,15 @@ export function toCartographicFromRadians(vector, cartographic) {
 export function toCartographicFromDegrees(vector, cartographic) {
   return toCartographic(vector, cartographic, config.cartographicRadians ? toRadians : noop);
 }
+
+export function isWGS84(vector) {
```

Comment:
Should we document this function? This checks that a coordinate is somewhere close to the surface of the earth on the earth ellipsoid?